R/concRespPlot2.R

```diff
@@ -35,7 +35,7 @@ concRespPlot2 <- function(row, log_conc = FALSE) {
     conc <- log10(conc)
     # replace the negative infinity with a number that is one log-10 unit
     # less than the second lowest dose (in log).
-    conc <- replace(conc, conc==-Inf, sort(conc)[2]-1)
+    conc <- replace(conc, conc==-Inf, sort(unique(conc))[2]-1)
     conc_plot <- seq(from=min(conc),to=max(conc),length=100)
     conc_plot <- replace(conc_plot, conc_plot==-Inf, sort(conc)[2]-1)
```

Comment:
Does this line of code also need to be updated to ensure we are using the appropriate x-range?